app/views/shared_users/index.html.erb

```diff
@@ -26,7 +26,7 @@
 									<%= shared.email %>
 								</td>
 								<td>
-            			<%= link_to "/appointments?user_check=#{shared.id}" do %>
```

Comment:
Tem certeza dessa modificação?

Reply:
I fixed it here Diego, thanks for the hello 👍🏽